### ISSUE TITLE
What with default route from dhcp to virtualized network?

### ISSUE THREAD

**Issue by hex-dec** [microsoft/SDN#439]
We have VMM. We installed network controller with slb/mux and gateway vm.
When vm with virtualized network get ip from vmswitch dhcp there are not ip default gateway. 
Is this the way it should be, or is there a problem with something? I can't find the problem. There are no errors either.

**Comment by BuggeXX** [comment]
default getway ip is allways the first aviable address from the subnet

**Comment by hex-dec** [comment]
> default getway ip is allways the first aviable address from the subnet

Yes it`s. But dhcp from VMswitch does not send it. Why? Users must manually specify the default route or add unnecessary scripts to add it?
